tools/jslint/jslint.js

```diff
@@ -607,7 +607,9 @@ var JSLINT = (function () {
             use_object: "Use the object literal notation {}.",
             use_or: "Use the || operator.",
             use_param: "Use a named parameter.",
-            used_before_a: "'{a}' was used before it was defined.",
+            used_before_a: "'{a}' was used before it was defined. (If you have" +
+                " added a new global, you need to add it to the whitelist in" +
+                " tools/jslist/check-all.js)",
```

Comment:
I thought I had removed this but am clearly not all sorted, git-wise. I did send the separate PR so I think that once that's merged, this will vanish.